spec/rubocop/cli/cli_options_spec.rb

```diff
@@ -346,9 +346,9 @@ class SomeCop < Cop
 
               remaining_range =
                 pending_cop_warning.length..-(inspected_output.length + 1)
-              pending_cops = output[remaining_range].split("\n")
+              pending_cops = output[remaining_range]
 
-              expect(pending_cops).to include(' - Style/SomeCop (N/A)')
+              expect(pending_cops).to include("Style/SomeCop: # (new in N/A)\n  Enabled: true")
```

Comment:
"new in N/A" sound quite odd, maybe remove it altogether?

Reply:
Yeah, was thinking the same